docs/source/system_reference_guide/custom-environments.ipynb

```diff
@@ -116,15 +116,23 @@
     "\n",
     "### Using custom environments in jupyter notebooks\n",
     "\n",
-    "To make your environment accessible in a Jupyter notebook, you need to register a kernel that has your environment. You can use the `ipykernel install` command for this, which means you must list `ipykernel` as a dependency in your configuration file. An example is in `env3.yml`, that creates an environment named `env3`. After creating it, you run the kernel registration command this way :\n",
+    "- List ipykernel as a dependency in the Conda environment YAML file (env3.yml) used to create the custom environment. Here's an example:\n",
```

Comment:
Slightly modified your change. Can we use that for the whole section ?

> The following instruction steps are for python kernels. 
> 1. Make sure `ipykernel` is listed as a dependency in your configuration file. An example that has such a dependency is `env3.yml`
> 2. Create your environment based on that configuration file
> 3. Install the environment as a kernel by running (parameter values follow the example mentioned)
> `python -m ipykernel install --user --name env3 --display-name "Python env3"`
>
> This command installs the environment as a kernel in Jupyter, making it accessible in the notebook with a displayed name "Python env3". 
> - Wait around 30 seconds and launch a new notebook. Among the kernel options, you > should see "Python env3" listed. Below you can see a screenshot that shows what this > step looks like.
> `![Register a kernel with a conda environment and launch a notebook with it](../_static> /launch_registered_kernel_custom_conda.png)`

In case you don't know, the last line that starts with `!` is to link to a png file stored in  this repository. This way, it gets rendered on the docs (see https://docs.maap-project.org/en/getting-started-pr-branch/system_reference_guide/custom-environments.html#Using-custom-environments-in-jupyter-notebooks). This particular png shows the notebook kernel selection step.